ui/src/components/connections/ConnectionsTable.tsx

```diff
@@ -220,6 +220,7 @@ export function ConnectionsTable() {
         <ConnectionDeleteModal
           id={connectionSelected}
           onClose={() => setConnectionSelected(undefined)}
+          onDelete={() => void fetchConnections()}
```

Comment:
These can just be

`onClose={setConnectionSelected}`
`onDelete={fetchConnections}`

?

Reply:
I should convert `onDelete` to promise

Reply:
So will you do that?

Reply:
We need to pass `undefined` to `setConnectionSelected ` so we can not inline it.

> We should convert `onDelete` to promise

I don't understand this, but I think this code is ok as it's.

Reply:
I prefer to keep it with just one type, so no, I'll leave it with the void

Reply:
Both have to be as they are